subsys/tfm/boards/board/RTE_Device.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2019 Arm Limited. All rights reserved.
+ * Copyright (c) 2020 Nordic Semiconductor ASA. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//-------- <<< Use Configuration Wizard in Context Menu >>> --------------------
+
+#ifndef __RTE_DEVICE_H
+#define __RTE_DEVICE_H
+
+#include <devicetree.h>
+#include <autoconf.h>
+
+#define UARTE(idx)			DT_NODELABEL(uart##idx)
+#define UARTE_PROP(idx, prop)		DT_PROP(UARTE(idx), prop)
+
+// <e> USART (Universal synchronous - asynchronous receiver transmitter) [Driver_USART0]
+// <i> Configuration settings for Driver_USART0 in component ::Drivers:USART
+#define   RTE_USART0                    1
+//   <h> Pin Selection (0xFFFFFFFF means Disconnected)
+//     <o> TXD
+#define   RTE_USART0_TXD_PIN            UARTE_PROP(0, tx_pin)
+//     <o> RXD
+#define   RTE_USART0_RXD_PIN            UARTE_PROP(0, rx_pin)
+//     <o> RTS
+#define   RTE_USART0_RTS_PIN            UARTE_PROP(0, rts_pin)
+//     <o> CTS
+#define   RTE_USART0_CTS_PIN            UARTE_PROP(0, cts_pin)
+//   </h> Pin Configuration
+// </e> USART (Universal synchronous - asynchronous receiver transmitter) [Driver_USART0]
+
+// <e> USART (Universal synchronous - asynchronous receiver transmitter) [Driver_USART1]
+// <i> Configuration settings for Driver_USART1 in component ::Drivers:USART
+#define   RTE_USART1                    1
+//   <h> Pin Selection (0xFFFFFFFF means Disconnected)
+#if defined(CONFIG_BOARD_NRF5340PDK_NRF5340_CPUAPPNS) || defined(CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPPNS)
+//     <o> TXD
+#define   RTE_USART1_TXD_PIN            25 // TODO: Add to devicetree
+//     <o> RXD
+#define   RTE_USART1_RXD_PIN            26 // TODO: Add to devicetree
+#else
```

Comment:
@anangl This UART is not present in the devicetree. Is there any chance of adding it?

Reply:
@anangl?